### PR TITLE
[ADI] Add idea.comments_count column

### PR DESCRIPTION
**Pull request by ottaviano**

### CODE REVIEW

File: src/Entity/IdeasWorkshop/Idea.php [hunk 1]
```diff
@@ -639,6 +642,29 @@ public function decrementVotesCount(int $increment = 1): void
         $this->votesCount -= $increment;
     }
 
+    /**
+     * @SymfonySerializer\Groups({"idea_list_read", "idea_read"})
+     */
+    public function getNewCommentsCount(): int
```
Comment (illusionOfParadise) [inline src/Entity/IdeasWorkshop/Idea.php]: Why did you add this method that returns `$commentsCount` value as `getCommentsCount `?

File: src/EntityListener/IdeaThreadListener.php [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+class IdeaThreadListener
+{
+    public function prePersist(Thread $thread): void
+    {
+        if ($thread->isEnabled()) {
+            $thread->getIdea()->incrementCommentsCount();
+        }
+    }
+
+//    public function preUpdate(Thread $thread, PreUpdateEventArgs $event): void
```
Comment (illusionOfParadise) [inline src/EntityListener/IdeaThreadListener.php]: Should be deleted if not used.

Comment (ottaviano) [reply src/EntityListener/IdeaThreadListener.php]: nop, it's used.

Comment (ottaviano) [reply src/EntityListener/IdeaThreadListener.php]: :trollface:
